### PR TITLE
Add azure_rm_managed_disk_info module filter by tags

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Add azure_rm_managed_disk_info module filter by tags.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

azure_rm_manageddisk_info
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by Fred-sun** [comment]
@haiyuazhang @mybayern1974 review_need

**Comment by Fred-sun** [comment]
@haiyuazhang ping to merge!

**Comment by Fred-sun** [comment]
@haiyuazhang @gavinfish review_need

**Comment by Fred-sun** [comment]
Ping to review and merger！

File: lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py [hunk 1]
```diff
@@ -228,7 +229,7 @@ def list_items_by_resource_group(self):
 
         results = []
         for item in response:
-            if self.has_tags(item.tags, self.tags):
+            if self.has_tags(item.tags, eval(self.tags)):
```
Comment (gavinfish) [inline lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py:232]: This change does not look good for me. Is it possible to change the type for tags at first?

**Review by gavinfish — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py]: Do you think I need to import the json module and convert the type before using it?

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py:232]: Do you think I need to import the json module and convert the type before using it?

Comment (gavinfish) [reply lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py:232]: The tags parameter should be parsed as list like below ? https://github.com/ansible/ansible/blob/b48a89682bf9d84f12110ba1bff125a6978375fd/lib/ansible/modules/cloud/azure/azure_rm_dnszone_info.py#L148

**Comment by Fred-sun** [comment]
This PR has been migrated to [#72](https://github.com/ansible-collections/azure/pull/72).

**Pull request closed without merging**